### PR TITLE
feat(data): add new programs (> 100 in total)

### PR DESCRIPTION
**Pull request by pierrecamilleri**
This PR adds roughly 60 additionnal programs.

This addition raised following changes : 

- new program nature "avantage fiscal", with associated field "Montant de l'avantage fiscal". JSON schema and Program page updatede accordingly. 
- new allowed operators

Observation : 
- Performance is soon going to be an issue

### CODE REVIEW

**Review by JulienParis — APPROVED** [review]
praise: nice and clean :)
note: I mainly reviewed the front-end components' changes I trust you on the `.py` script

File: packages/web/src/components/TeeProgramDetail.vue [hunk 1]
```diff
@@ -128,6 +128,18 @@
           :description="`${program[`montant du financement`]}`"
         />
       </div>
+
```
Comment (yvalentin) [inline packages/web/src/components/TeeProgramDetail.vue:131]: note: I think all this `v-if` can be refactor and simply (can be done later !)

**Review by yvalentin — APPROVED** [review]
praise: Nice work !

File: packages/web/src/components/TeeProgramDetail.vue (hunk 1 above)
Comment (pierrecamilleri) [reply packages/web/src/components/TeeProgramDetail.vue:131]: Yes indeed - I have not much experience in Vue so I'll leave it for later.

**Pull request merged**